src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -1497,16 +1497,16 @@ public boolean canPrequeue() throws IOException {
   @SuppressWarnings("ConstantConditions")
   @Override
   public BackplaneStatus backplaneStatus() throws IOException {
-    BackplaneStatus.Builder builder = BackplaneStatus.newBuilder();
     Set<String> executeWorkers = getExecuteWorkers();
     Set<String> storageWorkers = getStorageWorkers();
-    builder.addAllActiveExecuteWorkers(executeWorkers);
-    builder.addAllActiveStorageWorkers(storageWorkers);
-    builder.addAllActiveWorkers(Sets.union(executeWorkers, storageWorkers));
-    builder.setDispatchedSize(client.call(jedis -> state.dispatchedOperations.size(jedis)));
-    builder.setOperationQueue(state.operationQueue.status(client.call(jedis -> jedis)));
-    builder.setPrequeue(state.prequeue.status(client.call(jedis -> jedis)));
-    return builder.build();
+    return BackplaneStatus.newBuilder()
+        .addAllActiveExecuteWorkers(executeWorkers)
+        .addAllActiveStorageWorkers(storageWorkers)
+        .addAllActiveWorkers(Sets.union(executeWorkers, storageWorkers))
+        .setDispatchedSize(client.call(jedis -> state.dispatchedOperations.size(jedis)))
+        .setOperationQueue(state.operationQueue.status(client.call(jedis -> jedis)))
+        .setPrequeue(state.prequeue.status(client.call(jedis -> jedis)))
+        .build();
```

Comment:
Aside from the formatting change, I don't see any material modifications to this. Does BackplaneStatus now do something different?

Reply:
No, just style change.